### PR TITLE
29 delete record confirmation

### PR DESCRIPTION
**Pull request by rocar**
This PR added delete record confirmation and with “—yes” to bypass it, it is for issue https://github.com/dominikbraun/timetrace/issues/29

### CODE REVIEW

File: cli/delete.go [hunk 1]
```diff
@@ -83,3 +97,24 @@ func deleteRecordCommand(t *core.Timetrace) *cobra.Command {
...
+		if len(s) > 1 {
+			continue
+		}
+		if strings.Compare(s, "n") == 0 {
+			return false
+		} else if strings.Compare(s, "y") == 0 {
+			break
+		} else {
+			continue
+		}
```
Comment (dominikbraun) [inline cli/delete.go:117]:
```suggestion
		}
```

This can be removed 😄

Comment (rocar) [reply cli/delete.go]: 😊You are right.

File: cli/delete.go [hunk 2]
```diff
@@ -79,6 +79,7 @@ func deleteRecordCommand(t *core.Timetrace) *cobra.Command {
 				return
 			}
 
+			showRecord(record)
 			if !confirmed {
 				if !askForConfirmation() {
 					out.Err("Record NOT deleted")
```
Comment (dominikbraun) [inline cli/delete.go:85]:
```suggestion
					out.Info("Record NOT deleted.")
```

This is just a suggestion: If the user has been asked whether they want to delete the record, and they decide to abort the deletion, is it really an error? I'd probably use something like `out.Info` instead.

Everything else looks good to me! 👍

Comment (rocar) [reply cli/delete.go]: Agree with that, it's not an error

**Review by dominikbraun — APPROVED** [review]
LGTM 👍

**Comment by dominikbraun** [comment]
This PR is included in [release v0.3.1](https://github.com/dominikbraun/timetrace/releases/tag/v0.3.1).

**Pull request merged**